opus/import/obs_cassini_common_pds4.py

```
@@ -0,0 +1,459 @@
+    'METHON': 'METHONE',
+    'ROCK': 'ROCKS',
+    'STAR -- CW LEO': 'STAR',
+    'STAR -- ETA CAR': 'STAR',
+    '--': 'UNKNOWN',
+    'UNK': 'UNKNOWN',
+    'F RING': 'SATURN-FRING'
+}
+
+
+class ObsCassiniCommonPDS4(ObsCommonPDS4):
+    def __init__(self, *args, **kwargs):
```

Ruff (0.14.5)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)